guides/source/active_support_core_extensions.md

```diff
@@ -2565,7 +2565,7 @@ WARNING. If the collection is empty the root element is by default "nil-classes"
 
 The name of children nodes is by default the name of the root node singularized. In the examples above we've seen "contributor" and "object". The option `:children` allows you to set these node names.
 
-The default XML builder is a fresh instance of `Builder::XmlMarkup`. You can configure your own builder via the `:builder` option. The method also accepts options like `:dasherize` and friends, they are forwarded to the builder:
+The default XML Builder is a fresh instance of `Builder::XmlMarkup`. You can configure your own builder via the `:builder` option. The method also accepts options like `:dasherize` and friends, they are forwarded to the builder:
```

Comment:
I think this refers to _a_ XML builder, not the XML Builder template format or the class to delegate to. We also don't upcase `builder` in the following lines. So I think we should keep this as is.